Modules/Challenges/test/Challenges.API.Tests.Performance.EndToEnd/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "typescript",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "ssh://git@github.com/k6io/example-typescript.git",
+  "author": "Simon Aronsson <simme@k6.io>",
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/core": "7.13.16",
+    "@babel/plugin-proposal-class-properties": "7.13.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.13.8",
+    "@babel/preset-env": "7.13.15",
+    "@babel/preset-typescript": "7.13.0",
+    "@types/k6": "~0.41.0",
+    "@types/webpack": "5.28.0",
+    "babel-loader": "8.2.2",
+    "clean-webpack-plugin": "4.0.0-alpha.0",
+    "copy-webpack-plugin": "^9.0.1",
+    "k6-expect": "^0.0.15",
+    "typescript": "4.2.4",
+    "webpack": "^5.75.0",
+    "webpack-cli": "4.6.0",
+    "webpack-glob-entries": "^1.0.1"
+  },
+  "scripts": {
+    "get-test": "k6 run -e HOST=http://localhost:5000 -e VUS=1 -e ITERATIONS=100 -e CLIENT_SECRET=test dist/get.test.js",
+    "post-with-authentication": "k6 run -e HOST=http://localhost:5000 -e VUS=1 -e ITERATIONS=100 -e CLIENT_SECRET=test dist/post.with.authentication.test.js",
+    "post-without-authentication": "k6 run -e HOST=http://localhost:5000 -e VUS=1 -e ITERATIONS=100 dist/post.without.authentication.test.js",
+    "start": "webpack && npm run get-test && npm run post-with-authentication && npm run post-without-authentication"
```

Comment:
Maybe we should use shell/Powershell scripts instead of npm scripts. Because currently you need to change the npm script each time you want to run the test on a different HOST. And as far as I know it's not possible to pass arguments to npm scripts.
So instead, we provide a script called run-all.sh, which has an argument for the host, so you can execute it like:

```
./run-all.sh localhost:5000
```

I will update the Jira issue accordingly.